caffe2/operators/mean_op.cc

```diff
@@ -79,19 +79,21 @@ print("Mean:", workspace.FetchBlob("X"))
 class GetMeanGradient : public GradientMakerBase {
   using GradientMakerBase::GradientMakerBase;
   vector<OperatorDef> GetGradientDefs() override {
+    auto inputs = std::vector<string>();
```

Comment:
I don't think you need to change GetMeanGradient.

Reply:
See comment above.